testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/security/callerprincipal/GetCallerPrincipalTestCase.java

```diff
@@ -97,11 +95,6 @@
 
     public static final String LOCAL_USER = "$local";
 
-    @BeforeClass
-    public static void beforeClass() {
-        AssumeTestGroupUtil.assumeElytronProfileTestsEnabled();
```

Comment:
it looks GetCallerPrincipalTestCase is ok if I move "System.getProperty("elytron") == null" test outside of callable - if the condition should stay, the only failure is that system property is not visible from callable (which is minor)